Q and A/Q and A/CustomTableViewCell.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  CustomTableViewCell.swift
+//  Q and A
+//
+//  Created by Cameron Dunn on 1/15/19.
+//  Copyright © 2019 Cameron Dunn. All rights reserved.
+//
+
+import UIKit
+
+class CustomTableViewCell: UITableViewCell {
+    @IBOutlet weak var askedByLabel: UILabel!
+    @IBOutlet weak var questionLabel: UILabel!
+    @IBOutlet weak var infoLabel: UILabel!
+    var question : Question? = nil{
+        didSet{
+            updateViews()
+        }
+    }
+    private func updateViews(){
+        askedByLabel.text = question!.asker
+        questionLabel.text = question!.question
```

Comment:
Remove line 21-22. Because you're passing the object here and updating views based on the object's properties. First unwrap the `question` and assign the values
```suggestion
        guard let question = question else { return }
        questionLabel.text = question.question
        askedByLabel.text = question.asker
        if question.answer == nil {
            infoLabel.text = "Can you answer this?"
        } else {
            infoLabel.text = "Tap to view answer."
        }
```